### PR TITLE
Add hero 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the page is empty. There is no content nor sections. #17 

**What is the new behavior (if this is a feature change)?**
- A hero section has been added. #17

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Seems fine to me. 🌸

**Review by Aichnerc — COMMENTED** [review]
Still seems fine to me.

File: package.json [hunk 1]
```diff
@@ -1,15 +1,17 @@
 {
   "name": "your-app",
-  "version": "0.1.6",
+  "version": "0.1.7",
   "private": true,
   "// If You want to install mdbreact from our GitLab repository, please replace '4.18.0' with gitURL provided here ->": "git+https://oauth2:<YOUR-TOKEN-GOES-HERE>@git.mdbootstrap.com/mdb/react/re-pro.git",
```
Comment (kleberbaum) [inline package.json]: Documentation, comments, and docstrings should be wrapped at 79 characters.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
It should be ok to merge after a few changes.

File: package.json (hunk 1 above)
Comment (Aichnerc) [reply package.json]: This comment was useless and has been removed.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**